code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -587,6 +588,15 @@
 						chest.overlays += overlay
 					else
 						src.overlays += overlay
+				if(module.second_overlay)
```

Comment:
i'm concerned about this second overlay, it seems like a violation of general design principles.

The only numbers in computing are zero, one, and infinity. If you need more than one, you should support an arbitrary quantity